src/core/unit.c

```diff
@@ -3453,6 +3453,7 @@ int unit_patch_contexts(Unit *u) {
 
                         ec->private_tmp = true;
                         ec->remove_ipc = true;
+                        ec->private_devices = true;
```

Comment:
hmm, this appears to be a bug in the man page, we should drop that... It should only say this for PrivateTmp=...